website/src/content/docs/docs/extending-typespec/emitter-framework.md

```diff
@@ -3,369 +3,189 @@ id: emitter-framework
+The emitter framework has four separate but related areas:
 
-  // emit my entire TypeSpec program
-  assetEmitter.emitProgram();
-  // or, maybe emit types just in a specific namespace
-  const ns = context.program.resolveTypeReference("MyNamespace")!;
-  assetEmitter.emitType(ns);
+1. **alloy**: a framework for code generation that provides a React-like functional component model, manages symbols and references, renders and formats source text, and does other things useful regardless of what language you're emitting. [Alloy](https://alloy-project.github.io/alloy) is not part of TypeSpec, and we are developing it to be useful even if you aren't writing a TypeSpec emitter. [Read the documentation for the Alloy framework](https://alloy-framework.github.io/alloy/guides/getting-started/).
+2. **alloy language components**: Libraries of components for various languages. For example, allows for declaring a variable in TypeScript, and then creating a reference to that variable such that any necessary imports and package.json dependencies are added. [Read the documentation for these components](https://alloy-framework.github.io/alloy/guides/getting-started/).
+3. **typekits**: A convenient way to pull information from the TypeSpec type graph.
+4. **emitter framework**: builds off of alloy and its language components to provide TypeSpec-aware components and utilities. For example, while the alloy language component for TypeScript interfaces allows manually declaring members, the emitter framework component can take a TypeSpec data type and convert it directly into a TypeScript interface.
 
-  // lastly, write your emit output into the output directory
-  await assetEmitter.writeOutput();
-}
-```
+When you are writing a TypeSpec emitter, you will draw on components and APIs from each of these areas.
 
-### `TypeEmitter<T>`
+## How to get started
 
-This is the base class for writing logic to convert TypeSpec types into assets in your target language. Every TypeSpec type has at least one method on this base class, and many have multiple methods. For example, models have both `ModelDeclaration` and `ModelLiteral` methods to handle `model Pet { }` declarations and `{ anonymous: boolean }` literals respectively.
+We recommend you read the alloy documentation to understand its component model, reactivity system, formatting capabilities, and so forth. Alloy's [core concepts](https://alloy-framework.github.io/alloy/guides/basic-concepts/) document is a good place to start.
 
-To support emitting all TypeSpec types, you should expect to implement all of these methods. But if you don't want to support emitting all TypeSpec types, you can either throw or just not implement the method, in which case the type will not be emitted.
+Also, make sure to read the 'Getting Started' section under the [emitter basics](./emitters-basics.md) topic. To use the framework, you will need an emitter library and a `$onEmit` function.
```

Comment:
> 'Getting Started'

I think this section is currently named "Starting Out" - but I like "Getting Started" better - possible to update that file?